app/src/pages/CardSubmission/WaitingForSubmissions.tsx

```diff
@@ -27,87 +27,93 @@ function WaitingForSubmissions() {
+              <Grid item>
+                <Typography variant="h5">
+                  {submittedOrAcceptedSoFar}/{`${total} cards so far`}
+                </Typography>
+              </Grid>
+            </>
+          ),
+          [WaitingForSubmissionsState.SubmittedAssign]: (
             <Grid item>
-              {currentPlayer.role === PlayerRole.Host ? (
-                <Button
-                  size="large"
-                  variant="contained"
-                  color="primary"
-                  onClick={async () => {
-                    const players = teamsWithSequence(currentGame.players)
-                    await updateAllPlayers({
-                      variables: {
-                        gameId: currentGame.id,
-                        players: players.map(({ id, team, team_sequence }) => ({
-                          id,
-                          team,
-                          team_sequence,
-                        })),
-                      },
-                    })
-                    updateGameState({
-                      variables: {
-                        id: currentGame.id,
-                        state: GameStateEnum.TeamAssignment,
-                      },
-                    })
-                  }}
-                >
-                  Assign Teams
-                </Button>
-              ) : null}
+              All players submitted! As the host, you can now assign teams.
             </Grid>
-          )}
-        </>
-      ) : (
-        <>
-          <Grid item container justify="center">
-            Just waiting for everyone else...<div style={{ width: 4 }}></div>
-            {waitingForPlayers.map((player) => (
-              <>
-                <PlayerChip username={player.username || ""}></PlayerChip>
-                <div style={{ width: 4 }}></div>
-              </>
-            ))}
-          </Grid>
-          <Grid item>
-            <Typography variant="h5">
-              {submittedSoFar}/{`${total} cards so far`}
-            </Typography>
-          </Grid>
-          <Grid item>
-            <div style={{ marginTop: 50 }}>
-              <Fishbowl></Fishbowl>
-            </div>
-          </Grid>
-        </>
+          ),
+          [WaitingForSubmissionsState.SubmittedWait]: (
+            <Grid item>
+              {`All players submitted, ${submittedOrAcceptedSoFar} cards in total. Now we
+              are waiting on the host to start the game!`}
+            </Grid>
+          ),
+        }[state]
+      }
+
+      {PlayerRole.Host === currentPlayer.role && currentGame.screen_cards && (
+        <Grid item container direction="column" spacing={2} alignItems="center">
+          {unscreenedCards.map((card, index) => (
+            <Grid item key={index}>
+              <ScreenCard card={card}></ScreenCard>
+            </Grid>
+          ))}
+        </Grid>
       )}
+
+      {WaitingForSubmissionsState.SubmittedAssign === state ? (
+        <AssignTeamsButton />
```

Comment:
todo: disable this button if screening cards, and not screened all of them? Game might continue to work without screening them (Bc of how that current game subscription is written)

Reply:
Yeah, I think we could leave this enabled unless there's a reason to force the host to screen all of them explicitly.

Reply:
Cool let's leave it enabled

todo: some spacing so it doesn't jam up against the review cards

Reply:
<img width="340" alt="Screen Shot 2020-07-15 at 8 08 16 PM" src="https://user-images.githubusercontent.com/847532/87611826-04432900-c6d7-11ea-8038-141453324dd0.png">